vosekast_control/I2cInputAndRelayControl.py

```diff
@@ -0,0 +1,70 @@
+import smbus
+from typing import Dict, List
+
+
+class RelayControl:
+    def __init__(self):
+        self.address_relays = 0x38
```

Comment:
What, if we want to use a second relay with a other address. Maybe you can move that to the params with a default value.
And maybe you should only say `address` and not `address_relays`.